src/marshaller/HTML.js

```diff
@@ -1,15 +1,17 @@
 "use strict";
 (function (root, factory) {
     if (typeof define === "function" && define.amd) {
-        define(["d3", "../layout/Grid", "./HipieDDL", "../layout/Surface", "../layout/Cell"], factory);
+        define(["d3", "../layout/Grid", "./HipieDDL", "../layout/Surface", "../layout/Cell", "../layout/Popup"], factory);
     } else {
-        root.marshaller_HTML = factory(root.d3, root.layout_Grid, root.marshaller_HipieDDL, root.layout_Surface, root.layout_Cell);
+        root.marshaller_HTML = factory(root.d3, root.layout_Grid, root.marshaller_HipieDDL, root.layout_Surface, root.layout_Cell, root.layout_Popup);
     }
-}(this, function (d3, Grid, HipieDDL, Surface, Cell) {
+}(this, function (d3, Grid, HipieDDL, Surface, Cell, Popup) {
     function HTML() {
         Grid.call(this);
 
         this.surfacePadding(0);
+
+        this.flyoutWidgets = {};
```

Comment:
should i use a widgetarray ... i dont think this stuff should be serialized though ....